src/SamplesApp/SamplesApp.UITests/Constants.cs

```diff
@@ -14,6 +14,6 @@ public class Constants
 		public readonly static string AndroidAppName = "uno.platform.unosampleapp";
 
 		// Default active platform when running under Visual Studio test runner
-		public const Platform CurrentPlatform = Platform.Android;
+		public const Platform CurrentPlatform = Platform.Browser;
```

Comment:
not sure if this was intentionally committed... but I approve :p

Reply:
It's not intentional, but I think I'll change the way UI Tests are structured to have one per target, so we don't need to do that dance.